File: Android App/MasterKey-AndroidApp/app/src/main/java/com/example/masterkey/Models/SecretBox.java
package com.example.masterkey.Models;

public class SecretBox {
    private User user;
    private Accounts accounts;

    public SecretBox(User user) {
        this.user = user;
        this.accounts = new Accounts();
    }

    //Getters
    public User getUser() {
        return user;
    }

    public Accounts getAccounts() {
        return accounts;
    }
}

File: iOS App/MasterKey/MasterKey/Models/SecretBox.swift
//
//  SecretBox.swift
//  MasterKey
//
//  Created by Wembley on 5/4/22.
//

import Foundation

struct SecretBox: Codable {
    var user: User
    var accounts: Accounts
    
    init(user: User, accounts: Accounts) {
        self.user = user
        self.accounts = accounts
    }
}
